tests/em/static/test_DC_Utils.py

```diff
@@ -204,21 +204,6 @@ def test_ElecSep(self):
 
         self.assertTrue(passed)
 
-    def test_apparent_resistivity(self):
```

Comment:
Why was this removed?

Reply:
This may have been rendered redundant by creating another test